src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -320,6 +330,9 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
         const withdrawAmountCryptoBaseUnit = toBaseUnit(value, asset.precision)
         const amountCryptoBaseUnit = toBaseUnit(withdrawAmountCryptoPrecision, asset.precision)
 
+        if (withdrawAmountCryptoPrecision.gt(amountAvailableCryptoPrecision))
+          return 'common.insufficientFunds'
```

Comment:
Don't run the query and early bail if the withdraw amount is greater than the staked balance + rewards